src/flow/darcy_flow_mh.cc

```diff
@@ -454,7 +476,15 @@ void DarcyMH::update_solution()
     time_->next_time();
 
     time_->view("DARCY"); //time governor information output
+
+    solve_time_step();
+}
+
+
+void DarcyMH::solve_time_step(bool output)
```

Comment:
When this new output flag is used?
This whole function is prototype how to deal with equations with coefficients discontinuous in time. So it should be made part of the generic equation base class and used consistently at the top coupling level.

Reply:
This method is used either within a time iteration (then we want to output) or within HM iteration (then we do not output unless converged).